core/dbt/adapters/base/impl.py

```diff
@@ -655,9 +671,17 @@ def convert_agate_type(cls, agate_table, col_idx):
     ###
     # Operations involving the manifest
     ###
-    def execute_macro(self, manifest, macro_name, project=None):
+    def execute_macro(self, manifest, macro_name, project=None, context=None):
```

Comment:
`context` means a lot of different things in different places in dbt / jinja. can you rename `context` to something more descriptive? e.g. `context_overrides`